GoogleForCreators/web-stories-wp
# Pre-Publish Checklist: Update strings

## Summary

Updates the text in the pre-publish checklist. Using updated strings from [Huge suggestion document](https://docs.google.com/spreadsheets/d/1PjaRp4bueQ3Cv14A7I696m0vjGCIX_kN9tIGh-V-Bkc/edit?ts=5fdcc25f#gid=920793050). Sorry, this one is tedious to test.

## Relevant Technical Choices

none

## To-do

Once implemented, we should update the file to make the WIP tab the new official strings tab.

## User-facing changes

Pre publish checklist has updated text

## Testing Instructions

- trigger warnings or errors in the pre-publish checklist to see the new text

|Before|After|
|--|--|
|<image src="https://user-images.githubusercontent.com/22185279/104783448-465f2d00-5743-11eb-8c9f-0b80e0f56881.png" width="250px" />|<image src="https://user-images.githubusercontent.com/22185279/104783476-5545df80-5743-11eb-8863-2389a99d0a3d.png" width="250px" />|



---



Fixes #5848
